selfdrive/car/toyota/interface.py

```diff
@@ -261,6 +261,15 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
 
     return ret
 
+  # uncomment to disable radar:
+  # @staticmethod
```

Comment:
this is better than leaving around in a PR, but I'd merge it as actual code behind an `if False` or a flag so it survives refactors